README.md

```diff
@@ -396,18 +397,56 @@ authClient.signIn({
 
 ### `signOut()`
 
-Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example).
+Signs the user out of their current [Okta session](https://developer.okta.com/docs/api/resources/sessions) and clears all tokens stored locally in the `TokenManager`. The Okta session can be closed using either an `XHR` method or a `redirect` method. By default, the `XHR` method is used. Here are some points to consider when deciding which method is most appropriate for your application:
+
+XHR:
+
+* Executes in the background. The user will see not any change to `window.location`.
+* Will fail if 3rd-party cookies are blocked by the browser.
+* If the session is already closed, the method will fail and reject the returned Promise. This error can be caught by the app and handled seamlessly.
+* It is recommended (but not required) for the app to call `window.location.reload()` after the `XHR` method completes to ensure your app is properly re-initialized in an unauthenticated state.
+* See [Close Current Session](https://developer.okta.com/docs/reference/api/sessions/#close-current-session) in the Session API documentation.
+
+Redirect:
+
+* Will change the `window.location` to an Okta-hosted page before redirecting to a URI of your choice.
+* No issue with 3rd-party cookies.
+* Requires a `postLogoutRedirectUri` to be specified. This URI must be whitelisted in the Okta application's settings.
+* Requires a valid `idToken`. If the `idToken` is invalid or session does not exist the redirect may end on a 400 error page from Okta. This error will be visible to the user and cannot be handled by the app.
```

Comment:
Is this true? I thought only a non-whitelisted post_logout_redirect_uri gave errors

Reply:
Yeah, good catch. I see that an invalid session does not produce an error. I'll see if I can suss out any other error conditions.

Reply:
Documentation here lists some error conditions (including invalid id_token), I'll try to repro: https://developer.okta.com/docs/reference/api/oidc/#error-conditions